reana-ui/src/util.js

```diff
@@ -237,6 +237,34 @@ export function formatSearch(term) {
   return term ? JSON.stringify({ name: [term] }) : term;
 }
 
+/**
+ * Format file size with human-readable units.
+ * @param {Number} size File size in bytes.
+ * @returns The formatted human-readable file size.
+ */
+export function formatFileSize(size, digits = 2) {
+  if (size === 0) {
+    return "0 Bytes";
+  }
+
+  const units = [
+    "Bytes",
```

Comment:
very minor: what about removing the capital letter (`bytes`) or maybe using directly the symbol `B`?

Reply:
I also like `bytes` or `B` more, but I kept `Bytes` to be consistent with reana-db:
```python
    @staticmethod
    def _human_readable_bytes(bytes_):
        """Convert bytes usage to human readable string."""
        if bytes_ == 0:
            return "0 Bytes"
        units = ["Bytes", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"]
        ...
```